packages/cozy-harvest-lib/src/policies/clisk.js

```diff
@@ -11,7 +11,7 @@ import { LOGIN_SUCCESS_EVENT } from '../models/flowEvents'
  * @param {import('cozy-client/types/types').KonnectorsDoctype} konnector - konnector object
  * @returns {Boolean}
  */
-const isClisk = konnector => Boolean(konnector.clientSide)
+const isClisk = konnector => Boolean(konnector?.clientSide)
```

Comment:
Why this method can be called without having a konnector? We should fix the caller, not this method, no? Also, type & jsdoc have to been updated

Reply:
Yes, I fixed the call (in the FlowProvider) to test if there is a konnector